src/target/imxrt.c

```diff
@@ -227,6 +270,68 @@ bool imxrt_probe(target_s *const target)
 	return true;
 }
 
+static void imxrt_id_chip(target_s *target, imxrt_priv_s *priv)
+{
+	// The iMX RT series doesn't have a device id register. Instead, the NXP universal flash loader
+	// uses known ROM values at a particular address to differentiate the devices. That code uses
+	// three locations but only one location is actually needed.
+	// https://github.com/nxp-mcuxpresso/i.mxrt-ufl/blob/main/src/ufl_find_target.c
+	uint32_t rom_location = 0;
+	const uint16_t cpuid_partno = target->cpuid & CPUID_PARTNO_MASK;
+	if (cpuid_partno == CORTEX_M33)
+		rom_location = IMXRT_X00_ROM_FINGERPRINT_ADDR;
+	else if (cpuid_partno == CORTEX_M7)
+		if (target->part_id == 0x88c6U)
+			rom_location = IMXRT_11XX_ROM_FINGERPRINT_ADDR;
+		else
+			rom_location = IMXRT_10XX_ROM_FINGERPRINT_ADDR;
+	else
```

Comment:
```suggestion
	else if (cpuid_partno == CORTEX_M7) {
		if (target->part_id == 0x88c6U)
			rom_location = IMXRT_11XX_ROM_FINGERPRINT_ADDR;
		else
			rom_location = IMXRT_10XX_ROM_FINGERPRINT_ADDR;
	} else
```
You went a touch too far removing braces - this is considered a multi-line block so should still have them. It's single-line blocks that they get dropped from.